internal/rest/client/truststore.go

```diff
@@ -0,0 +1,25 @@
+package client
+
+import (
+	"context"
+	"time"
+
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/microcluster/internal/rest/types"
+)
+
+// AddTrustStoreEntry adds a new record to the truststore on all cluster members.
+func AddTrustStoreEntry(ctx context.Context, c *Client, args types.ClusterMemberLocal) error {
+	queryCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
+	defer cancel()
+
+	return c.QueryStruct(queryCtx, "POST", InternalEndpoint, api.NewURL().Path("truststore"), args, nil)
+}
+
+// DeleteTrustStoreEntry deletes the record corresponding to the given cluster member from the trust store.
+func DeleteTrustStoreEntry(ctx context.Context, c *Client, args types.ClusterMemberLocal) error {
```

Comment:
Not sure about the other MicroCluster endpoints in regards to consistency, but would it make sense to have a `DELETE /cluster/internal/truststore/<member>` endpoint instead?

Reply:
I am thinking about the `PUT/DELETE /cluster/1.0/cluster/<member>` endpoints that use the same concept.

Reply:
Done.